docs/_snippets/nextjs-app-directory-in-meta.md

```diff
@@ -11,8 +11,8 @@ export default {
 };
 ```
 
-```ts filename="NavigationBasedComponent.stories.ts" renderer="react" language="ts-4-9"
-import { Meta, StoryObj } from '@storybook/react';
+```ts filename="NavigationBasedComponent.stories.ts" renderer="react" language="ts"
+import type { Meta, StoryObj } from '@storybook/nextjs-vite';
```

Comment:
I think this would be better, given that `nextjs-vite` isn't the default:

```suggestion
import type { Meta, StoryObj } from '@storybook/nextjs'; // Or nextjs-vite
```